core/deployment/src/main/java/io/quarkus/deployment/steps/NativeImageConfigBuildStep.java

```diff
@@ -91,33 +84,14 @@ void build(SslContextConfigurationRecorder sslContextConfigurationRecorder,
 
             if (graalVmHome != null) {
                 Path graalVmLibDirectory = Paths.get(graalVmHome, "jre", "lib");
-                Path linuxLibDirectory = graalVmLibDirectory.resolve("amd64");
-                Path linuxPath = linuxLibDirectory.resolve(LIB_SUN_EC);
 
                 // We add . as it might be useful in a containerized world
                 javaLibraryPathAdditionalPath.produce(new JavaLibraryPathAdditionalPathBuildItem("."));
-                if (Files.exists(linuxPath)) {
-                    // On Linux, the SunEC library is in jre/lib/amd64/
-                    // This is useful for testing or if you have a similar environment in production
-                    javaLibraryPathAdditionalPath
-                            .produce(new JavaLibraryPathAdditionalPathBuildItem(linuxLibDirectory.toString()));
-                } else {
-                    // On MacOS, the SunEC library is directly in jre/lib/
-                    // This is useful for testing or if you have a similar environment in production
-                    javaLibraryPathAdditionalPath
-                            .produce(new JavaLibraryPathAdditionalPathBuildItem(graalVmLibDirectory.toString()));
-                }
 
                 // This is useful for testing but the user will have to override it.
                 sslTrustStoreSystemProperty.produce(
                         new SslTrustStoreSystemPropertyBuildItem(
                                 graalVmLibDirectory.resolve(Paths.get("security", "cacerts")).toString()));
```

Comment:
I suppose we want to keep this around. I'll need it to be confirmed.

Reply:
Yes.